client/extensions/woocommerce/app/order/order-activity-log/events.js

```diff
@@ -0,0 +1,125 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize, moment } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { keys, last, sortBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isActivityLogLoaded,
+	getActivityLogEvents,
+} from 'woocommerce/state/sites/orders/activity-log/selectors';
+import OrderEvent from './event';
+import OrderEventsByDay from './day';
+
+function getSortedEvents( events ) {
+	const eventsByDay = {};
+	events.forEach( event => {
+		const day = moment( event.timestamp ).format( 'YYYYMMDD' );
+		if ( eventsByDay[ day ] ) {
+			eventsByDay[ day ].push( event );
+		} else {
+			eventsByDay[ day ] = [ event ];
+		}
+	} );
+
+	keys( eventsByDay ).forEach( day => {
+		eventsByDay[ day ] = sortBy( eventsByDay[ day ], [ 'timestamp', 'key' ] ).reverse();
+	} );
+	return eventsByDay;
+}
+
+class OrderEvents extends Component {
+	static propTypes = {
+		orderId: PropTypes.number.isRequired,
+		siteId: PropTypes.number.isRequired,
+	};
+
+	componentWillMount() {
+		this.setState( { openDay: last( keys( this.props.eventsByDay ) ) } );
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		const newOpenDay = last( keys( nextProps.eventsByDay ) );
+		//if a new latest day has been appended, open it
+		if ( ! this.props.eventsByDay[ newOpenDay ] ) {
+			this.setState( { openDay: newOpenDay } );
+		}
+	}
+
+	toggleOpenDay = date => {
+		this.setState( () => ( { openDay: date } ) );
+	};
+
+	renderNotes = () => {
+		const { days, eventsByDay, translate } = this.props;
+		if ( ! days.length ) {
+			return <p>{ translate( 'No activity yet' ) }</p>;
```

Comment:
Hi! I've found a possible matching string that has already been translated **84** times:
`translate( 'No activity yet!' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wcs-ui-compatibility)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).